processor/workers.go

```diff
@@ -397,6 +435,9 @@ func CountStats(fileJob *FileJob) {
 						return
 					}
 				}
+				if Trace {
```

Comment:
The `printTrace` function already checks `if Trace`, so it's redundant to have it here as well.

Reply:
Oh its because if you don't include it the fmt.sprintf kills performance. The trace/verbose/debug should be refactored to pass those in to avoid that issue. Its on my list that I may get around to one of these days.